components/cdc/src/errors.rs

```diff
@@ -26,6 +27,8 @@ pub enum Error {
     Mvcc(MvccError),
     #[fail(display = "Request error {:?}", _0)]
     Request(ErrorHeader),
+    #[fail(display = "Engine traits error {}", _0)]
+    EngineTraits(EngineTraitsError),
```

Comment:
Can TiCDC handle this new error?

Reply:
For now we reflect these errors to `RegionNotFound`. 😅